docs/ug/getting_started.md

```diff
@@ -49,8 +49,18 @@ export CC="gcc-9.2.0"
 export CXX="g++-9.2.0"
 export LLVM_BINROOT="/usr/pack/riscv-1.0-kgf/pulp-llvm-0.12.0/bin"
 # As a temporary workaround (until correct tool versions are installed system-wide):
-export PATH=/home/colluca/snitch/bin:$PATH
 export PATH=/usr/scratch/dachstein/colluca/opt/verible/bin:$PATH
+
+# Install spike for spike-dasm (submodule of the repository)
+cd sw/deps/riscv-isa-sim
+autoconf
+mkdir build
+cd build
+../configure
+make
```

Comment:
Installation instructions provided do not terminate successfully

Reply:
To keep track of debugging:
It seems to work with GCC version `8.5.0` and `4.8.5`, but not with `9.2.0`

Running `make` with version `9.2.0` fails with the following error:
```
g++-9.2.0 -MMD -MP  -Wall -Wno-unused -g -O2 -std=c++11 -DPREFIX=\"/usr/local\" -I. -I.. -I../fesvr -I../riscv -I../dummy_rocc -I../softfloat -I../spike_main -x c++-header ../riscv/insn_template.h -o insn_template.h.gch
/lib/../lib64/crt1.o: In function `_start':
(.text+0x20): undefined reference to `main'
collect2: error: ld returned 1 exit status
make: *** [insn_template.h.gch] Error 1
```